dev_guide/compute_resources.adoc

```diff
@@ -70,27 +229,48 @@ A compute resource is classified with a quality of service based on the specifie
 
-A container may have a different quality of service for each compute resource. For example, a container can have *Burstable* CPU and *Guaranteed* memory qualities of service.  
+A container may have a different quality of service for each compute resource.
+For example, a container can have *Burstable* CPU and *Guaranteed* memory
+qualities of service.
 
-The quality of service has an impact on what happens if the resource is compressible or not. CPU is a compressible resource, whereas memory is an incompressible resource.
+The quality of service has an impact on what happens if the resource is
+compressible or not. CPU is a compressible resource, whereas memory is an
+incompressible resource.
 
 With CPU Resources: ::
-- A *BestEffort* CPU container is able to consume as much CPU as is available on a node with the lowest priority.
-- A *Burstable* CPU container is guaranteed to get the minimum amount of CPU requested, but it may or may not get additional CPU time. Excess CPU resources are distributed based on the amount requested across all containers on the node.
-- A *Guaranteed* CPU container is guaranteed to get the amount requested and no more, even if there are additional CPU cycles available. This provides a consistent level of performance independent of other activity on the node.
+- A *BestEffort* CPU container is able to consume as much CPU as is available on a
+node with the lowest priority.
+- A *Burstable* CPU container is guaranteed to get the minimum amount of CPU
+requested, but it may or may not get additional CPU time. Excess CPU resources
+are distributed based on the amount requested across all containers on the node.
+- A *Guaranteed* CPU container is guaranteed to get the amount requested and no
+more, even if there are additional CPU cycles available. This provides a
+consistent level of performance independent of other activity on the node.
 
 With Memory Resources: ::
-- A *BestEffort* memory container is able to consume as much memory as is available on the node, but the scheduler is subject to placing that container on a node with too few memory to meet a need. In addition, a *BestEffort* memory container has the greatest chance of being killed if there is an out of memory event on the node.
-- A *Burstable* memory container is scheduled on the node to get the amount of memory requested, but it may consume more. If there is an out of memory event on the node, *Burstable* containers are killed after *BestEffort* containers when attempting to recover memory.
-- A *Guaranteed* memory container gets the amount of memory requested, but no more. In the event of an out of memory event, it will only be killed if there are no more *BestEffort* or *Burstable* memory containers on the system.
+- A *BestEffort* memory container is able to consume as much memory as is
+available on the node, but the scheduler is subject to placing that container on
+a node with too few memory to meet a need. In addition, a *BestEffort* memory
```

Comment:
s/few/little/ (to me, memory has more of a continuous quantity feel than discrete quantity (the reality)).  Also, "to meet a need" is probably completely superfluous.